File: cpp/1036.cpp
#include <stdio.h>
#include <cmath>

using namespace std;

int main()
{
    double a, b, c, delta;
    scanf("%lf %lf %lf", &a, &b, &c);
    delta = b * b - 4 * a * c;

    if (a == 0 || delta < 0) {
        printf("Impossivel calcular\n");
    } else {
        printf("R1 = %.5f\n", (-b + sqrt(delta)) / (2 * a));
        printf("R2 = %.5f\n", (-b - sqrt(delta)) / (2 * a));
    }

    return 0;
}

File: py/1036.py
a, b, c = map(float, input().split())
delta = b * b - (4 * a * c)

if delta < 0 or a == 0:
    print('Impossivel calcular')
else:
    R1 = (-b + (delta ** (1/2))) / (2 * a)
    R2 = (-b - (delta ** (1/2))) / (2 * a)

    print('R1 =', round(R1, 5))
    print('R2 =', round(R2, 5))